pages/common/topgrade.md

```diff
@@ -0,0 +1,28 @@
+# Topgrade
+
+> Update all your applications.
+> More at <https://github.com/r-darwish/topgrade>.
+
+- Call updates:
```

Comment:
Perhaps "Run updates" sounds better?

Reply:
I agree. Will fix these changes in a few hours.